pkg/services/ngalert/models/alert_rule.go

```diff
@@ -335,6 +338,7 @@ type AlertRuleVersion struct {
 	For         time.Duration
 	Annotations map[string]string
 	Labels      map[string]string
+	IsPaused    bool
```

Comment:
I wonder if a better name for this field would be `Enabled`? @grobinson-grafana wdyt?

Reply:
I think is better to name it so that the default value is the same as the type's default value, which is `false` in this case. This allows us to make fewer code changes and default behavior comes free of cost. I also like to name booleans starting with `Is` or `Has`.

We could name it `IsDisabled` but IMO `IsPaused` is better understood.

Reply:
I agree with Alex. I also prefer it when the default value is the same.